docs/Reference/APIs/PrivacyEnhancementsAPIs.md

```diff
@@ -0,0 +1,30 @@
+# Privacy Enhancements  APIs
+## APIs
+### `getContractPrivacyMetadata` 
+The api to query the Privacy Metadata for a contract account address
+#### Parameter
+* contract address
+
+#### Returns
+* `creationTxHash`: ACOTH
+* `privacyFlag`: unsigned integer with `1` for PP, `3` for PSV contracts (default value is `0`).
+
+#### Examples
+
+```jshelllanguage tab="JSON RPC"
+// Request
+curl -X POST http://127.0.0.1:22000 --data '{"jsonrpc":"2.0","method":"eth_getContractPrivacyMetadata","params":["0x1932c48b2bf8102ba33b4a6b545c32236e342f34"],"id":15}' --header "Content-Type: application/json"
```

Comment:
try not to mix different code in the same block. Here you have `curl` command which is a shell command call and the return value that can be displayed as json.